Dockerfile

```diff
@@ -43,6 +45,13 @@ COPY scripts/rbspy_build.sh .
 RUN ./rbspy_build.sh
 RUN mv "/tmp/rbspy/target/$(uname -m)-unknown-linux-musl/release/rbspy" /tmp/rbspy/rbspy
 
+# dotnet-trace
+FROM mcr.microsoft.com/dotnet/sdk${DOTNET_BUILDER_UBUNTU} as dotnet-builder
+RUN apt-get update && \
+  dotnet tool install --global dotnet-trace
```

Comment:
Not all versions of packages/programs are pinned. We do pin the base Docker images, and most importantly we pin the profiler versions. Since we currently download dotnet-trace install of building it - make sure to download a pinned version from `dotnet tool install` if possible (to request a specific version and preferably to verify its hash against what you expect to receive here)

Reply:
I'm not sure, but dotnet-trace version may be linked to the dotnet version - in this case 6.0. I purposedly have different versions of dotnet on my machine and in other dotnet containers that I test it on. It seems to work properly on all of them.